components/_patterns/02-molecules/date-bars/_date-bar.scss

```diff
@@ -0,0 +1,23 @@
+.date-bar {
+  align-items: center;
+  background-color: $color-success;
+  color: $white;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 1em;
+  font-weight: 600;
+  list-style-type: none;
+  min-height: 80px;
+  padding: $space-s $space;
+
+  &__date {
+    @include heading-small($color: $white);
```

Comment:
I'm not sure this is the right thing to use here... When I was poking around, I disabled half of the styles coming from this mixin, and it turns out they were duplicates, or just the default behaviors.

I think the headings need to be validated and probably renamed to make sense in this project before we start using them in other components.

In fact, I was playing, and if these are just divs (like I'm going to suggest in a minute) instead of h2s, we don't need _any_ of these styles to accomplish the same end design

Reply:
After getting into it, I want to talk it through before I suggest removing the `h2`s... I think I'm just missing some information

Reply:
The request to use `h2`s comes from @wlla based on how this component works in the larger page—`h1` being the event title, `h2` containing this information for the event. Since this is coming from her I think it's best for her to weigh in since there are outstanding concerns over it.